MovingBlocks/Terasology
# chore(JNA): update JNA to 5.6.0,  remove com.sun.jna from jopenvr.jar

### Contains

Updating JNA
Removing embeded jna from jopenvr.jar
Fixes #4151
Fixes #4146

### How to test

1. ~~Check what VR works~~ :D (Seems, It is not works many time...)
2.  Check Terasology that works on Liberica 11 on Clean Windows (default libraries only)

### Outstanding before merging
